codec-http2/src/main/java/io/netty/handler/codec/http2/DefaultHttp2Connection.java

```diff
@@ -131,18 +131,24 @@ final boolean isClosed() {
             promise.trySuccess(null);
             return promise;
         }
+
         Iterator<PrimitiveEntry<Http2Stream>> itr = streamMap.entries().iterator();
         // We must take care while iterating the streamMap as to not modify while iterating in case there are other code
         // paths iterating over the active streams.
         if (activeStreams.allowModifications()) {
```

Comment:
Perhaps "concurrent" was a poor choice of words in this context... see https://github.com/netty/netty/pull/5205#discussion_r62297682